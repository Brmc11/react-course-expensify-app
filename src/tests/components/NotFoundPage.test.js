import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/NotFoundPage';

test('should render a fo oh fo when shit aint found', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
