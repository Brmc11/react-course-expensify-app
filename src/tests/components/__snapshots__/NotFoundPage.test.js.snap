// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a fo oh fo when shit aint found 1`] = `
<div>
  FO OH FO bitch -
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
